src/css/profile/mobile/common/popup.less

```diff
@@ -202,16 +199,16 @@
 
 	.ui-popup-notitle {
 		@media (orientation: portrait) and (max-width: 479px) {
-			margin-top: 24 * @px_base;
```

Comment:
In case of, we don't need to check the max-width?
Doesn't it have a different value only depending on the orientation mode?

Reply:
agree. I modified.

done.